scripts/lib/build-utils.ts

```diff
@@ -294,6 +294,22 @@ export default {
     });
   },
 
+  /**
+   * Build Privileged Service.
+   */
+  async buildPrivilegedService(): Promise<void> {
+    const parentDir = path.join(this.rootDir, 'resources', 'win32', 'bin');
```

Comment:
@mook-as Initially I placed it in internal, however, the reason behind putting it in `bin` is because of the installer only adds `bin` to the path [here](https://github.com/rancher-sandbox/rancher-desktop/blob/3fa97f49763dcb67dae7091e69635973b31acac7/build/add-to-path.ps1#L17).

Reply:
The installer doesn't need to use `PATH`, it has the full path to the executable on disk.  In fact, it _shouldn't_ use `PATH`; if there's a different `privileged-service.exe` earlier, we _don't_ want to run that one. (It's a rather generic name, that is mostly identified by the path on disk.)